packages/datatrak-web/src/layout/UserMenu/UserInfo.tsx

```diff
@@ -5,58 +5,31 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { Typography } from '@material-ui/core';
+import { Box, Typography } from '@material-ui/core';
 import { RouterLink } from '@tupaia/ui-components';
-import { Button } from '../../components';
+import { Button, ChangeProjectButton } from '../../components';
 import { useCurrentUser } from '../../api';
 import { ROUTES } from '../../constants';
 
-const Wrapper = styled.div`
-  padding-left: 1rem;
+const Wrapper = styled(Box)`
+  padding-inline-start: 1rem;
   ${({ theme }) => theme.breakpoints.up('md')} {
-    padding-left: 1.5rem;
+    padding-inline-start: 1.5rem;
   }
 `;
 
-const Details = styled.div`
-  align-items: center;
+const Details = styled(Box)`
```

Comment:
Out of interest, is there any particular reason for using `Box` as opposed to just a `div`?

Reply:
Tom had mentioned off-handedly that if there’s a MUI component, then it’s probably preferable. To be completely honest, I’m not sure what `Box` enables other than a higher CSS selector specificity (i.e. its generated class name)

If you have a preference for plain old `div`s, I’m more than happy to do that. (Is it more performant at build time?)